inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -18,14 +19,13 @@
                                        batch_replace_params_in_template,
                                        replace_params_in_template)
 from inductiva.utils import files
-from inductiva.utils.files import remove_files_with_tag
 from inductiva.fluids.scenarios.wind_tunnel.post_processing import WindTunnelSimulationOutput
 from inductiva.tasks import Task
 
 SCENARIO_TEMPLATE_DIR = os.path.join(TEMPLATES_PATH, "wind_tunnel")
 OPENFOAM_TEMPLATE_INPUT_DIR = "openfoam"
 FILES_SUBDIR = "files"
-COMMANDS_TEMPLATE_NAME = "commands.json.jinja"
+COMMANDS_TEMPLATE_FILE_NAME = "commands.json.jinja"
```

Comment:
Do you think it is necessary to define it as a global variable? It will be the same for all the simulators, right?

Reply:
Some simulators don't require commands, so it can't be general for all simulators. However, I think this and other functionalities (like the function to replace parameters in a template commands file) should be somewhere where they could be shared. I am still thinking where that could go, so I think I'll leave it for another PR!